### PR TITLE
Introduces StatsId and colibri changes around it

### PR DESCRIPTION
**Pull request by damencho**
Changes colibri to use <endpoint and stops using <channel-bundle. Uses statsId if available.

### CODE REVIEW

**Comment by bgrozev** [comment]
LGTM! Does it need an update of pom.xml?

**Comment by damencho** [comment]
Yep, it needs update for the jitsi.desktop maven artifact.

File: src/test/java/org/jitsi/jicofo/ColibriTest.java [hunk 1]
```diff
@@ -113,13 +113,13 @@ public void testChannelAllocation()
 
         ColibriConferenceIQ peer1Channels
             = colibriConf.createColibriChannels(
-                peer1UseBundle, peer1, true, contents);
-
+                peer1UseBundle, peer1, null, true, contents);
+        System.err.println("bbbbbbb  " + peer1Channels.toXML());
```
Comment (bgrozev) [inline src/test/java/org/jitsi/jicofo/ColibriTest.java]: woops

Comment (damencho) [reply src/test/java/org/jitsi/jicofo/ColibriTest.java]: yep, woops. will update it with the pom

**Pull request merged**